githubscript/docker.d/worker.yml

```diff
@@ -42,26 +42,25 @@ github_projects:
         github_repo_name: staging-android-components
         contact_github: true
     'COT_PRODUCT == "xpi" && ENV == "prod"':
-      xpi-manifest:
+      mozilla-extensions/xpi-manifest:
         allowed_actions: ["release"]
         github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_PROD" }
         github_owner: mozilla-extensions
         github_repo_name: xpi-manifest
         contact_github: true
+        allow_github_repo_override: false
```

Comment:
This will probably be `true` when we roll out to production, so we can push releases to the various addons repos.